.idea/modules.xml

```diff
@@ -5,6 +5,7 @@
       <module fileurl="file://$PROJECT_DIR$/dist/plugins/DefaultPlugin/DefaultPlugin.iml" filepath="$PROJECT_DIR$/dist/plugins/DefaultPlugin/DefaultPlugin.iml" />
       <module fileurl="file://$PROJECT_DIR$/dist/plugins/PluginBase/PluginBase.iml" filepath="$PROJECT_DIR$/dist/plugins/PluginBase/PluginBase.iml" />
       <module fileurl="file://$PROJECT_DIR$/RoddyCore/RoddyCore.iml" filepath="$PROJECT_DIR$/RoddyCore/RoddyCore.iml" />
+      <module fileurl="file://$PROJECT_DIR$/../RoddyInfra/RoddyToolLib/RoddyToolLib.iml" filepath="$PROJECT_DIR$/../RoddyInfra/RoddyToolLib/RoddyToolLib.iml" />
```

Comment:
Arg, I answered on another page. Now my comment is somewhere... anyways. We can discuss this tomorrow.